src/helpers/ContractSynchronizer.ts

```diff
@@ -5,15 +5,64 @@ import getOwnedERC721, {
   parseLogData,
 } from 'helpers/getOwnedERC721'
 
+export interface ContractSynchronizerSchema {
+  account: string
+  synchronizedBlockId: number
+  addressToTokenIds: { [address: string]: string[] }
+}
+
 export default class ContractSynchronizer {
   account: string
   locked = false
   synchronizedBlockId?: number
   addressToTokenIds: { [address: string]: Set<string> } = {}
   skipTransactions = new Set<string>()
 
-  constructor(account: string) {
+  constructor(
+    account: string,
+    addressToTokenIds: { [address: string]: Set<string> } = {},
+    synchronizedBlockId?: number
+  ) {
     this.account = account
+    this.synchronizedBlockId = synchronizedBlockId
+    this.addressToTokenIds = addressToTokenIds
+  }
+
+  static fromJSON({
```

Comment:
Let's name it `contractSynchronizerFromJson`

Reply:
I see no reason to add this before the method name, this is a similar method as `toString`, `toNumber`, `toObject`